LICENSE.mrb_float_read

```diff
@@ -0,0 +1,20 @@
+This license terms applies to the mrb_read_float() function in
```

Comment:
I am going to replace `mrb_float_read()` with the public domain `strtod` implementation soon (probably today). So we don't need this file.

Reply:
I updated this PR.